docs/can-guides/setup/setting-up-canjs.md

```diff
@@ -933,19 +933,19 @@ With CanJS downloaded or installed, use it to create an `index.html` page with a
 This build only includes CanJS’s [can-core] and [can-infrastructure] modules and all of CanJS’s named exports are available on the [can-namespace can] object. For example, if you want the [can-ajax] infrastructure module, use it like `can.ajax`:
 
 ```js
-can.Component.extend({
-	tag: "my-app",
-	view: `
-        CanJS {{feels}} modules.
-        The server says: {{messagePromise.value}}
-    `,
-	ViewModel: {
-		feels: { default: "😍" },
-        messagePromise: {
-            default: () => can.ajax({url: "/message"})
-        }
-	}
-});
+class MyApp extends can.StacheElement {
```

Comment:
Forgot to look at the other changes before approving. Does `can.StacheElement` work?

Reply:
I'll try. I actually wanted to tag you on this since I wasn't sure.

Reply:
It does not work, `StacheElement` is not attached to the `can` object.

Reply:
I'm sure it's not working. We'll need to add can-namespace to the new packages and add these to it. We can merge this PR for now and fix that after.